vector/src/main/java/fr/gouv/tchap/activity/TchapRoomCreationActivity.java

```diff
@@ -166,13 +175,36 @@ private void hideKeyboard() {
         imm.toggleSoftInput(InputMethodManager.SHOW_FORCED, 0);
     }
 
-    @OnClick(R.id.rly_hexagon_avatar)
-    void addRoomAvatar() {
+    private void openMediasPicker() {
         Intent intent = new Intent(TchapRoomCreationActivity.this, VectorMediasPickerActivity.class);
         intent.putExtra(VectorMediasPickerActivity.EXTRA_AVATAR_MODE, true);
         startActivityForResult(intent, REQ_CODE_UPDATE_ROOM_AVATAR);
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.M)
+    @OnClick(R.id.rly_hexagon_avatar)
+    void addRoomAvatar() {
+        if (ContextCompat.checkSelfPermission(this, Manifest.permission.CAMERA) == PackageManager.PERMISSION_DENIED) {
```

Comment:
Why don't you use `CommonActivityUtils.checkPermissions(CommonActivityUtils.PERMISSION_CAMERA, this);` like in "VectorRoomDetailsActivity"?